packages/nouns-webapp/vite.config.ts

```diff
@@ -5,12 +5,11 @@ import { lingui } from '@lingui/vite-plugin';
 import { nodePolyfills } from 'vite-plugin-node-polyfills';
 import svgr from 'vite-plugin-svgr';
 import Inspect from 'vite-plugin-inspect';
-import tailwindcss from '@tailwindcss/vite'
+import checker from 'vite-plugin-checker';
 
 // https://vite.dev/config/
 export default defineConfig({
   plugins: [
-    tailwindcss(),
     react({
```

Comment:
was the tailwind plugin removed by accident?

Reply:
Nope, not fully integrated yet. Tailwind v4 needs a plugin, so I downgraded to v3 for now. v4 is still pretty new and has some compatibility quirks. Felt like it’s out of scope for this PR anyway; figured we can handle it separately if we ever ditch modular CSS and go full Tailwind.